website/docs/ref/enums.doc.js

```diff
@@ -0,0 +1,78 @@
+/* @flow */
+/*
+---
+id: enums
+title: Enums
+permalink: /docs/enums.html
+prev: react.html
+---
+*/
+
+/*
+In Flow you can [represent enums using literal types](builtins.html#literal-types).
+*/
+
+type Month = 'jan' | 'feb' | 'mar' | 'apr' | 'jun' | 'jul' | 'aug' | 'sep' | 'oct' | 'nov' | 'dic'
+
+const jan: Month = 'jan';
+// $ExpectError
+const wrong: Month = 'wrong'; // 'wrong' is not a Month
+
+/*
+This is very handy, but sometimes you need to access the enum definition at runtime (i.e. at a value level).
+
+Suppose for example that you want to associate a value to each month of the previous example.
+
+You could do
+*/
+
+const monthNumbers = {
+  jan: 1,
+  feb: 2,
+  mar: 3,
+  apr: 4,
+  may: 5,
+  jun: 6,
+  jul: 7,
+  aug: 8,
+  sep: 9,
+  oct: 10,
+  nov: 11,
+  dec: 12
```

Comment:
I... I don't remember pasting that.

The original comment I was typing was:

It would be good to have a shorter example than this, it currently takes up a lot of vertical space in the post and after the third or so item its redundant.